Tuist/Config.swift

```diff
@@ -0,0 +1,7 @@
+import ProjectDescription
+
+let config = Config(
+    generationOptions: [
+        .autogeneratedSchemes(.disabled),
```

Comment:
I don't think I agree with applying this setting. If the autogenerated schemes don't work for us, they are probably not apt for everybody else. I'd rather rethink the current autogeneration instead.

I believe the tuist project should use the features provide as much as possible.

Reply:
I agree the autogenerated schemes as they are now are not very useful as they are way too many.
I think we should in some way try to add the concept of module it tuist, but it's way out of the scope of this PR, which just want to make working with tuist a bit less painful 😁

Reply:
Agree with both of you! I think we should merge this in the meantime and create a ticket to update the generated schemes in `3.0`.

At work we also disable the generated schemes which is unfortunate as manually making the schemes is not super fun. We should come up with a nicer implementation that can work for the majority of projects.

Reply:
Yep I will try to write a proposal about that 👍

Reply:
[Here it is!](https://github.com/tuist/tuist/issues/3975)